Misc/NEWS

```diff
@@ -907,6 +907,10 @@ Build
 Tools/Demos
 -----------
 
+- bpo-24037: Argument Clinic now uses the converter `bool(accept={int})` rather
+  of `int` for semantical booleans.  This allows to not repeat the default
+  value for Python and C and will help in converting to `bool` in future.
```

Comment:
rather of → rather than
This allows to not repeat . . . → This avoids repeating . . . [see earlier changes I made regarding “allows to”: <https://bugs.python.org/issue26304>]